### ISSUE TITLE
Add support for writing out Geoshape as GeoJSON in GraphSON

### ISSUE THREAD

**Issue by BrynCooke** [thinkaurelius/titan#954]
Once https://github.com/thinkaurelius/titan/pull/952 is merged then we can consume GeoJSON in GraphSON, so it would be nice to have round trip support.

I think this just involves overriding

```
public GraphSONWriter.Builder graphSONWriter();
```

And adding a serializer for the Geoshape class.

**Comment by dalaro** [comment]
This sounds reasonable.  The idea is to have a GraphSONWriter subclass that adds a Geoshape serializer (that resides in the Titan codebase) to harmonize with #952?

**Comment by BrynCooke** [comment]
Yes this us exactly it. I think it will be straight forward to achieve.

**Comment by BrynCooke** [comment]
Plus it will resolve https://github.com/thinkaurelius/titan/issues/953

**Comment by mbroecheler** [comment]
@BrynCooke Please implement a special GraphSon extension for Titan to test out that TinkerPop method. Coordinate with @spmallette if you have any questions / issues.
